### PR TITLE
[hi] Added new glossary files in hindi docs , from archtecture tag to extension tag

### LINKED ISSUE #44962: [hi] Localize glossary in Hindi
**This is a Feature Request**

**What would you like to be added**
Many of the glossary elements are still not localize to the Hindi, their description is still in English.

**Why is this needed**
Many PRs that are open recently seems to be using glossary elements that are in English.

### PR DESCRIPTION
**Pull request by utkarsh-singh1**
Description:

Localised glossary contents in Hindi.

fixes #44962

### CODE REVIEW

**Comment by utkarsh-singh1** [comment]
related page - https://kubernetes.io/hi/docs/reference/glossary/?extension=true&core-object=true&community=true&architecture=true

File: content/hi/docs/reference/glossary/approver.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Approver
```
Comment (sftim) [inline content/hi/docs/reference/glossary/approver.md:2]: I'd expect to see Hindi here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/container-lifecycle-hooks.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Container Lifecycle Hooks
```
Comment (sftim) [inline content/hi/docs/reference/glossary/container-lifecycle-hooks.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/contributor.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Contributor
```
Comment (sftim) [inline content/hi/docs/reference/glossary/contributor.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/controller.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: Controller
```
Comment (sftim) [inline content/hi/docs/reference/glossary/controller.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/downstream.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: Downstream (disambiguation)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/downstream.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/downward-api.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+title: Downward API
```
Comment (sftim) [inline content/hi/docs/reference/glossary/downward-api.md:2]: You can use Hindi here if you prefer; if you do, you should use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/group-version-resource.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+---
+title: Group Version Resource
```
Comment (sftim) [inline content/hi/docs/reference/glossary/group-version-resource.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English terms.

File: content/hi/docs/reference/glossary/logging.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+---
+title: Logging
```
Comment (sftim) [inline content/hi/docs/reference/glossary/logging.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/managed-service.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+---
+title: Managed Service
```
Comment (sftim) [inline content/hi/docs/reference/glossary/managed-service.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/member.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+---
+title: Member
```
Comment (sftim) [inline content/hi/docs/reference/glossary/member.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/mixed-version-proxy.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
+---
+title: Mixed Version Proxy (MVP)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/mixed-version-proxy.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/network-policy.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
+---
+title: Network Policy
```
Comment (sftim) [inline content/hi/docs/reference/glossary/network-policy.md:2]: I'd expect to see “NetworkPolicy” here; the words “Network” and “Policy” aren't Hindi.

File: content/hi/docs/reference/glossary/pod-security-policy.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+---
+title: Pod Security Policy
```
Comment (sftim) [inline content/hi/docs/reference/glossary/pod-security-policy.md:2]: I'd write PodSecurityPolicy.

File: content/hi/docs/reference/glossary/quantity.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
+---
+title: Quantity
```
Comment (sftim) [inline content/hi/docs/reference/glossary/quantity.md:2]: You could write Hindi here; if you do, use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/reviewer.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
+---
+title: Reviewer
```
Comment (sftim) [inline content/hi/docs/reference/glossary/reviewer.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/service-catalog.md [hunk 16]
```diff
@@ -0,0 +1,16 @@
+---
+title: Service Catalog
```
Comment (sftim) [inline content/hi/docs/reference/glossary/service-catalog.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/sig.md [hunk 17]
```diff
@@ -0,0 +1,19 @@
+---
+title: SIG (special interest group)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/sig.md:2]: - Consider moving “special interest group” to the `aka` section.
- Consider putting a translation of “special interest group” into the `aka` section as well.

File: content/hi/docs/reference/glossary/toleration.md [hunk 18]
```diff
@@ -0,0 +1,17 @@
+---
+title: Toleration
```
Comment (sftim) [inline content/hi/docs/reference/glossary/toleration.md:2]: Optionally, use Hindi  here; you can then use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/upstream.md [hunk 19]
```diff
@@ -0,0 +1,18 @@
+---
+title: Upstream (disambiguation)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/upstream.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/volume-plugin.md [hunk 20]
```diff
@@ -0,0 +1,17 @@
+---
+title: Volume Plugin
```
Comment (sftim) [inline content/hi/docs/reference/glossary/volume-plugin.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/wg.md [hunk 21]
```diff
@@ -0,0 +1,18 @@
+---
+title: WG (working group)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/wg.md:2]: - Consider moving “working group” to the `aka` section.
- Consider putting a translation of “working group” into the `aka` section as well.

**Review by sftim — COMMENTED** [review]
We've an opportunity to add more Hindi to these entries.

**Comment by utkarsh-singh1** [comment]
Hi @sftim , updated PR , PTAL.

File: content/hi/docs/reference/glossary/cloud-controller-manager.md [hunk 22]
```diff
@@ -0,0 +1,18 @@
+---
+title: Cloud Controller Manager
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cloud-controller-manager.md:2]: Translate this, please

File: content/hi/docs/reference/glossary/configmap.md [hunk 23]
```diff
@@ -0,0 +1,16 @@
+---
+title: ConfigMap
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/configmap.md:2]: Translate this please

File: content/hi/docs/reference/glossary/event.md [hunk 24]
```diff
@@ -0,0 +1,22 @@
+---
+title: Event
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/event.md:2]: Translate this please

File: content/hi/docs/reference/glossary/gateway.md [hunk 25]
```diff
@@ -0,0 +1,18 @@
+---
+title: Gateway API
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/gateway.md:2]: This needs to be in Hindi

File: content/hi/docs/reference/glossary/ingress.md [hunk 26]
```diff
@@ -0,0 +1,18 @@
+---
+title: Ingress
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/ingress.md:2]: This needs to be in Hindi

File: content/hi/docs/reference/glossary/kube-controller-manager.md [hunk 27]
```diff
@@ -0,0 +1,17 @@
+---
+title: kube-controller-manager
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/kube-controller-manager.md:2]: This needs to be in Hindi

File: content/hi/docs/reference/glossary/kube-scheduler.md [hunk 28]
```diff
@@ -0,0 +1,17 @@
+---
+title: kube-scheduler
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/kube-scheduler.md:2]: This needs to be in Hindi

File: content/hi/docs/reference/glossary/kubernetes-api.md [hunk 29]
```diff
@@ -0,0 +1,17 @@
+---
+title: Kubernetes API
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/kubernetes-api.md:2]: This needs to be in Hindi

File: content/hi/docs/reference/glossary/qos-class.md [hunk 30]
```diff
@@ -0,0 +1,21 @@
+---
+title: QoS Class
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/qos-class.md:2]: Please check other localisations & see if this has been translated/transliterated to their respective language. Kindly do the same with this, as well.

File: content/hi/docs/reference/glossary/secret.md [hunk 31]
```diff
@@ -0,0 +1,19 @@
+---
+title: Secret
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/secret.md:2]: This should be in Hindi

File: content/hi/docs/reference/glossary/service-account.md [hunk 32]
```diff
@@ -0,0 +1,17 @@
+---
+title: ServiceAccount
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/service-account.md:2]: This should be in Hindi

File: content/hi/docs/reference/glossary/service.md [hunk 33]
```diff
@@ -0,0 +1,21 @@
+---
+title: Service
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/service.md:2]: This should be in Hindi

File: content/hi/docs/reference/glossary/sig.md [hunk 34]
```diff
@@ -0,0 +1,19 @@
+---
+title: SIG
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/sig.md:2]: This should be in Hindi

File: content/hi/docs/reference/glossary/spec.md [hunk 35]
```diff
@@ -0,0 +1,19 @@
+---
+title: Spec
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/spec.md:2]: This should be in Hindi

File: content/hi/docs/reference/glossary/taint.md [hunk 36]
```diff
@@ -0,0 +1,17 @@
+---
+title: Taint
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/taint.md:2]: This should be in Hindi

File: content/hi/docs/reference/glossary/volume.md [hunk 37]
```diff
@@ -0,0 +1,19 @@
+---
+title: Volume
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/volume.md:2]: This should be in Hindi

File: content/hi/docs/reference/glossary/wg.md [hunk 38]
```diff
@@ -0,0 +1,18 @@
+---
+title: WG
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/wg.md:2]: Please check how other localisations have translated/transliterated it & I'd advise the same be affected here.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/event.md (hunk 24 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/event.md:2]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/ingress.md (hunk 26 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/ingress.md:2]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/kube-controller-manager.md (hunk 27 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/kube-controller-manager.md:2]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/kube-scheduler.md (hunk 28 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/kube-scheduler.md:2]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/secret.md (hunk 31 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/secret.md:2]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/service-account.md (hunk 32 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/service-account.md:2]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/service.md (hunk 33 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/service.md:2]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/network-policy.md (hunk 12 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/network-policy.md]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/pod-security-policy.md (hunk 13 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/pod-security-policy.md]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/configmap.md (hunk 23 above)
Comment (sftim) [reply content/hi/docs/reference/glossary/configmap.md:2]: You can use `aka:` to also provide the original name.

File: content/hi/docs/reference/glossary/wg.md (hunk 38 above)
Comment (utkarsh-singh1) [reply content/hi/docs/reference/glossary/wg.md:2]: WG is termed as WG in most of the other translations.

File: content/hi/docs/reference/glossary/qos-class.md (hunk 30 above)
Comment (utkarsh-singh1) [reply content/hi/docs/reference/glossary/qos-class.md:2]: QoS is termed as QoS in many languages, only class has been translated.

**Comment by utkarsh-singh1** [comment]
@sftim @divya-mohan0209 PR has been updated , PTAL.

**Comment by utkarsh-singh1** [comment]
@sftim @divya-mohan0209, Is there any changed needed in this PR or is it ready for merge.

**Pull request closed without merging**